### ISSUE TITLE
Default settings

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/pdfjs#1]
Is there a way to create a page to set defaults rather than to require settings specified in each field display? (e.g. height, width, show page thumbnails by default or not, etc.)

**Comment by PepeMty** [comment]
Hey there..!
This issue has been here for 5+ years.
Any hope these config/options can be available...?

**Comment by laryn** [comment]
Yes, I am now a maintainer here and am interested in getting these options. It's just a matter of finding the time to do so -- but happy to review a PR if someone else wants to get this in sooner.

I'm thinking:
 - an admin configuration page for the defaults
 - an option in the per-instance configuration to use defaults (maybe a checkbox) and if it's not selected, allow overrides.
